app/models/ontology.rb

```diff
@@ -195,6 +195,23 @@ def versioned_locid
     current_version.locid
   end
 
+  def file_deleted?(commit_oid = nil)
+    !has_file?(commit_oid)
+  end
+
+  # alias_method doesn't work for this one.
```

Comment:
If we use `alias_method` instead, we get this error:

> The single-table inheritance mechanism failed to locate the subclass: 'SingleOntology'. This error is raised because the column 'type' is reserved for storing the class in case of inheritance. Please rename this column if you didn't intend it to be used for storing the inheritance class or overwrite Ontology.inheritance_column to use another column for that information.

So, I assume `alias_method` is bad for the combination STI + aliasing an `attr_accessible`.